tests/prepare.sh

```diff
@@ -0,0 +1,16 @@
+
+echo "[+] Install requirements ..."
+sudo apt update
```

Comment:
Wouldn't it be better/cleaner to execute this script as root instead of adding all the sudos in the script?